core/src/components/select/select.tsx

```diff
@@ -576,7 +586,7 @@ export class Select implements ComponentInterface {
       label = this.getLabel();
       labelText = label ? label.textContent : null;
     } else {
-      labelText = this.label;
+      labelText = this.labelText;
```

Comment:
Shouldn't this be `this.labelText()`?

Reply:
No, `this.labelText` is a getter which binds the property to a function. When you look up `this.labelText` the function is automatically called: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get